MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include CHANGELOG.md LICENSE README.md
+
+graft scripts
+graft tests
+graft docs
```

Comment:
Should the rst be included?

Reply:
I guess you could see the raw docs as part of the package's source distribution. I've copied the MANIFEST.in file from [Flask](https://github.com/pallets/flask/blob/master/MANIFEST.in), they also have their raw docs in it.

Reply:
Ok, if they do it then let's follow suit.